photon-ml/src/main/scala/com/linkedin/photon/ml/model/Coefficients.scala

```diff
@@ -50,13 +50,18 @@ protected[ml] case class Coefficients(means: Vector[Double], variancesOption: Op
     stringBuilder.toString()
   }
 
+  // TODO: what should we do about the epsilon used in this comparison?
   override def equals(that: Any): Boolean = {
+
+    def cmp(a1: Vector[Double], a2: Vector[Double]): Boolean =
+      (a1.toArray zip a2.toArray).foldLeft(true) { case (v, (m1, m2)) => v & math.abs(m2 - m1) < 1e-6}
```

Comment:
Done. For the record, persisting models and reloading them does introduce small variations, which I have observed and which is why I needed an epsilon. Previously written tests used "0" for all coefficients, which hid this issue (there was no error on "0").